Gemfile.lock

```diff
@@ -132,6 +132,7 @@ GEM
     bcrypt (3.1.20)
     bcrypt_pbkdf (1.1.1)
     bcrypt_pbkdf (1.1.1-arm64-darwin)
+    bcrypt_pbkdf (1.1.1-x64-mingw-ucrt)
```

Comment:
I'll be interested to see if this resumes being problematic when any of these gems update...